scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -78,16 +79,16 @@ _Note: Contributions intended for the official library shall not include the man
   - The optional paste coverage multiplier determines how much of the exposed copper area is covered by paste. (`EP_paste_coverage`) {float (0..1), default=0.65}
 
 ### Rounding of exposed pad features
-IPC excludes exposed pads from the requirement for rounding its corners. By default the exposed pad does therefore not use rounded corners. Some datasheets do however suggest the use of rounded corners either specified to a specific value or implicitly shown to be equal to the normal pads.
+IPC excludes exposed pads from the requirement for rounding its corners. By default the exposed pad does therefore not use rounded corners. Some datasheets do however suggest the use of rounded corners either specified to a specific value or they appear to be equal to the normal pads.
```

Comment:
The equality isn't given, even if it is implied. Suggest changing the ending of this line to '...they appear to be equal in corner radius to the normal pads'. You could also replace 'do however suggest the use of' with 'show' for clarity since a datasheet proving an image of a footprint with rounded EP corners isn't suggesting it.

Reply:
fixed